### ISSUE TITLE
SQS transport doesn't support Python 3

### ISSUE THREAD

**Issue by felixonmars** [celery/kombu#662]
```
    def get_all_queues(self, prefix=""):
        if not prefix:
            keys = sorted(self.queues.keys())[:1000]
        else:
            keys = filter(
                lambda k: k.startswith(prefix), sorted(self.queues.keys())
>           )[:1000]
E           TypeError: 'filter' object is not subscriptable
```

**Comment by ask** [comment]
This code is not in master?

**Comment by felixonmars** [comment]
I assume this has been fixed in #674. Sorry for the late response!
